features/step_definitions/admin_steps.rb

```diff
@@ -59,6 +59,8 @@ def update_settings(settings)
 Given /^basic languages$/ do
   Language.default
   german = Language.find_or_create_by_short_and_name("DE", "Deutsch")
+  german.support_available = true
+  german.save!
```

Comment:
It should be possible to add this to line 61:  `german = Language.find_or_create_by_short_and_name_and_support_available("DE", "Deutsch", true)`